Makefile

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2019-2022 Alibaba Cloud. All rights reserved.
+# Copyright (c) 2019-2022 Ant Group. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+default: build
+
+build:
+	# FIXME: This line will be removed when we solve the vm-memory dependency problem in Dragonball Sandbox
+	cargo update -p vm-memory:0.8.0 --precise 0.7.0 || true
+	cargo update -p fuse-backend-rs --precise 0.9.4 || true
+	cargo update -p vmm-sys-util --precise 0.9.0 || true
```

Comment:
remove `cargo update`
Currently, this is no longer required to control the version.

Reply:
For generated binary projects, The `Cargo.lock` file can be submitted.
See [Why do binaries have Cargo.lock in version control, but not libraries?](https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries) for details.